.gitlab-ci.yml

```diff
@@ -0,0 +1,34 @@
+include: 'https://raw.githubusercontent.com/Nitrokey/common-ci-jobs/master/common_jobs.yml'
+
+stages:
+  - pull-github
+  - build
+  - deploy
+
+variables:
+  #Repo for shared scripts (pull.sh release.sh, nightly_upload.sh):
+  GIT_STRATEGY: clone            #This seems to have no effect also set in webinterface
+  GIT_DEPTH: 0                    #This seems to have no effect also set in webinterface
+  GIT_SUBMODULE_STRATEGY: recursive #This seems to have no effect also set in webinterfac
+  SCRIPTS_REPO: git@git.dotplex.com:nitrokey/gitlab-ci.git
+  REPO_USER: nitrokey
+  REPO_NAME: nitrokey-storage-firmware
+
+build:
+  image: nk-storage-firmware-build:latest
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "push"'
+    - if: '$CI_PIPELINE_SOURCE == "schedule"'
+    - if: '$CI_PIPELINE_SOURCE == "web"'
+  tags:
+    - lxc
+  stage: build
+  script:
+    - cd src && make build-docker-toolchain
+    - make docker-make
+  after_script:
+    - cp ./src/*.hex artifacts
+    - wget 188.40.174.112:8089/checkmark/$CI_COMMIT_REF_NAME/$CI_COMMIT_SHA/$CI_JOB_NAME/$CI_JOB_STATUS/${CI_JOB_URL#*/*/*/}
```

Comment:
Is this IP documented somewhere? Why IP instead of a domain (not futureproof, as IPs are changing)?